src/components/Kv/KvMapLibre.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+	<div class="tw-w-full tw-relative">
+		<div :id="`kv-map-holder-${this.mapId}`" :ref="refString" style="min-height: 320px; min-width: 320px;">
+			<h1>a map</h1>
```

Comment:
Since these maps are decorative, can we remove this h1 and text?

Reply:
def will remove.

Reply:
Replaced with default black image panel...it's kinda harsh though...